### PR TITLE
zh-TW: cr2 Android translations

### PR DESCRIPTION
**Pull request by henrif75**
Professional translations for CR2 content.
Changes are being staged in a separate branch until all zh-TW content is reviewed.

#1463 #684

### CODE REVIEW

File: po/zh-TW.po [hunk 1]
```diff
@@ -11741,20 +11778,28 @@ msgid ""
...
 "The [Android Interface Definition Language (AIDL)](https://developer.android."
 "com/guide/components/aidl) is supported in Rust:"
 msgstr ""
+"Rust 支援 [Android 介面定義語言 (AIDL)](https://developer.android.com/guide/"
+"components/aidl)："
 
 #: src/android/aidl.md
 msgid "Rust code can call existing AIDL servers,"
-msgstr ""
+msgstr "Rust 程式碼可以呼叫現有的 AIDL 伺服器。"
```
Comment (victorhsieh) [inline po/zh-TW.po:11798]:
```suggestion
msgstr "Rust 程式碼可以呼叫現有的 AIDL 服務。"
```

File: po/zh-TW.po [hunk 2]
```diff
@@ -11741,20 +11778,28 @@ msgid ""
...
 
 #: src/android/aidl.md
 msgid "You can create new AIDL servers in Rust."
-msgstr ""
+msgstr "您可以在 Rust 中建立新的 AIDL 伺服器。"
```
Comment (victorhsieh) [inline po/zh-TW.po:11802]:
```suggestion
msgstr "您可以在 Rust 中建立新的 AIDL 服務。"
```

File: po/zh-TW.po [hunk 3]
```diff
@@ -12427,8 +12492,9 @@ msgid ""
 msgstr ""
 
 #: src/android/testing.md
+#, fuzzy
 msgid "Testing in Android"
-msgstr ""
+msgstr "在 Android 中建構"
```
Comment (victorhsieh) [inline po/zh-TW.po:12497]:
```suggestion
msgstr "在 Android 中的測試"
```

**Review by victorhsieh — APPROVED** [review]

**Comment by henrif75** [comment]
@victorhsieh Thank you!

**Pull request merged**